UnityProject/Assets/Scripts/Tilemaps/Behaviours/Meta/Atmospherics/Data/GasMix.cs

```diff
@@ -29,10 +29,10 @@ public class GasMix
 		public float Temperature
 		{
 			get { return temperature; }
-			set { temperature = Math.Clamp(value, 0, Single.MaxValue); }
+			set { temperature = Math.Clamp(value, 0.001f, Single.MaxValue); }
```

Comment:
explain

Reply:
I'm thinking maybe something tries to go 
n / temperature